main.js

```diff
@@ -49,4 +53,123 @@ const mobilMenu = function menu() {
+    sourcesLink: 'https://github.com/Nicolaswg/Nicolas_Microverse_Portfolio',
+  },
+  {
+    title: 'Tonic',
+    img: 'src/portfolio_img_c.PNG',
+    description: 'Lorem ipsum, dolor sit amet consectetur adipisicing elit. Possimus inventore nulla doloribus quia! Ea dolor, sequi reprehenderit architecto eum magnam molestiae dolore quae officia, minus voluptatum animi reiciendis qui illo ipsam nisi soluta culpa? Incidunt, minus, eaque dolorem doloremque enim tempore ipsum debitis dignissimos recusandae laborum sit harum sequi quibusdam provident officia nam odit vitae illum magni laudantium soluta voluptatem laborum sit harum sequi quibusdam provident officia nam odit vitae illum magni laudantium soluta voluptatem.',
+    pageLink: 'https://nicolaswg.github.io/Nicolas_Microverse_Portfolio/',
+    sourcesLink: 'https://github.com/Nicolaswg/Nicolas_Microverse_Portfolio',
+  },
+  {
+    title: 'Multi-Post Stories',
+    img: 'src/portfolio_img_d.PNG',
+    description: 'Lorem ipsum, dolor sit amet consectetur adipisicing elit. Possimus inventore nulla doloribus quia! Ea dolor, sequi reprehenderit architecto eum magnam molestiae dolore quae officia, minus voluptatum animi reiciendis qui illo ipsam nisi soluta culpa? Incidunt, minus, eaque dolorem doloremque enim tempore ipsum debitis dignissimos recusandae laborum sit harum sequi quibusdam provident officia nam odit vitae illum magni laudantium soluta voluptatem laborum sit harum sequi quibusdam provident officia nam odit vitae illum magni laudantium soluta voluptatem.',
+    pageLink: 'https://nicolaswg.github.io/Nicolas_Microverse_Portfolio/',
+    sourcesLink: 'https://github.com/Nicolaswg/Nicolas_Microverse_Portfolio',
+  }];
+// Modal projects event
+// First Modal
+projectBtn.forEach((btn, index) => {
+  btn.addEventListener('click', () => {
+    printModal(`<h2 class="modal-title">${modalProject[index].title}</h2>
+      <i id ="close-menu" class="fas fa-times"></i>
+      <ul class="modal-mark">
+        <li>
+          <p>CANOPY</p>
+        </li>
+        <li>
+          <p>Back End Dev</p>
+        </li>
+        <li>
+          <p>2015</p>
+        </li>
+      </ul>
+      <div class="modal-img">
+        <img src="${modalProject[index].img}" alt="modal img">
+      </div>
+      <p class="modal-description">${modalProject[index].description}</p>
+      <ul class="modal-skills">
+        <li>
+          <p href="">html</p>
+        </li>
+        <li>
+          <p href="">css</p>
+        </li>
+        <li>
+          <p href="">javascript</p>
+        </li>
+      </ul>
+      <div class="modal-buttons">
+        <div class="modal-btn">
+          <a href="${modalProject[index].pageLink}">See live</a>
+          <i class="fas fa-external-link-alt"></i>
+        </div>
+        <div class="modal-btn">
+          <a href="${modalProject[index].sourcesLink}">See Source</a>
+          <i class="fab fa-github"></i>
+        </div>
```

Comment:
- [ ] Kindly have elements on the page aligned and have spacing and padding matching with the figma design, Currently buttons and description of pop-up is overlapped with the image as shown below:
#### Your project (Desktop view)
![image](https://user-images.githubusercontent.com/61377322/146319687-38f94b44-7847-4c21-9419-e2f27a853bd9.png)

#### Your project (Mobile view)
![image](https://user-images.githubusercontent.com/61377322/146320787-9016ee28-01fe-4dc7-b189-8d6f4ffc17e5.png)

#### Figma design
![image](https://user-images.githubusercontent.com/61377322/146319703-be8905c6-6600-4e80-8dae-7efc0e1330c4.png)

> Hint: Using `position: absolute` to align elements is tricky and makes elements on the page less responsive. It is advised to use `flex-box` and `grids` to align elements. Please refer here for [more](https://css-tricks.com/quick-whats-the-difference-between-flexbox-and-grid/).